packages/tkg/bundle/config/values.yaml

```diff
@@ -86,6 +86,10 @@ tkrSourceControllerPackage:
     skipVerifyRegistryCert: false
     initialDiscoverFrequency: 60
     continuousDiscoverFrequency: 600
+    tkgProxyCaCert:
+    tkgCustomImageRepositoryCaCert:
+    tkgCustomImageRepository:
+    defaultImageRepository:
     deployment:
```

Comment:
I don't think these values (which would be part of configmap ) would be used in TKR Source controller as of now. The custom Image repository path is already factored while setting the values for `tkrRepoImagePath` ,`bomMetadataImagePath` ,`bomImagePath` by CLI. We can add these values later if needed

Reply:
done

Reply:
discussed with @12345lcr , `imageRepository` is consumed by tanzucli and addon controller as well in legacy mode. added it back to avoid potential incompatible issue during upgrade. cc @imikushin